liquibase/src/main/java/io/micronaut/liquibase/endpoint/RanChangeSetSerializer.java

```diff
@@ -1,64 +0,0 @@
-/*
- * Copyright 2017-2020 original authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.micronaut.liquibase.endpoint;
-
-import com.fasterxml.jackson.core.JsonGenerator;
-import com.fasterxml.jackson.databind.JsonSerializer;
-import com.fasterxml.jackson.databind.SerializerProvider;
-import io.micronaut.context.annotation.Type;
-import jakarta.inject.Singleton;
-import liquibase.changelog.RanChangeSet;
-
-import java.io.IOException;
-import java.time.Instant;
-
-/**
- * Jackson serializer for {@link RanChangeSet}.
- *
- * @author Iván López
- * @since 1.0.0
- */
-@Singleton
-@Type(RanChangeSet.class)
-public class RanChangeSetSerializer extends JsonSerializer<RanChangeSet> {
```

Comment:
Removing a public class is a breaking change. I doubt that someone is using it... but :shrug: